src/pytest_inmanta_lsm/remote_orchestrator.py

```diff
@@ -286,7 +286,6 @@ def setup_config(self) -> None:
         """
         Setup the config required to make it possible for the client to reach the orchestrator.
         """
-        inmanta_config.Config.load_config()
```

Comment:
I think this will fail if the project fixture hasn't been loaded at all, because there is no config loaded at all. 
So you need to check here if it has been loaded...

Reply:
I don't think so, because of 
- https://github.com/inmanta/inmanta-core/blob/d25205bdd49016596ad7653597a2cc99a8ed3992/src/inmanta/config.py#L230
- https://github.com/inmanta/inmanta-core/blob/d25205bdd49016596ad7653597a2cc99a8ed3992/src/inmanta/config.py#L162
- https://github.com/inmanta/inmanta-core/blob/d25205bdd49016596ad7653597a2cc99a8ed3992/src/inmanta/config.py#L154